### DISCUSSION TITLE
What's Left for 5.0

### DISCUSSION THREAD

**Discussion by popeyeotaku** [jkotlinski/durexforth#575] [General]
What's left for the next release of durex? It's supposed to be more standards compliant, right? I'd love to contribute if we could get the issues organized and a bit of a plan together

**Comment by jkotlinski** [comment]
Hi!
It would probably be fine to release what is currently on master. Maybe
pull request #564 should be merged first.

tis 19 nov. 2024 kl. 04:56 skrev popeyeotaku ***@***.***>:

**Comment by popeyeotaku** [comment]
Did the block wordset ever get finished?

**Reply by jkotlinski** [reply]
In case you are interested in contributing to the block wordset specifically, I could try refreshing my understanding of it and try to write down what is left to do there.

**Comment by jkotlinski** [comment]
It is a work in progress, except I don’t work on it :-)

tis 19 nov. 2024 kl. 07:20 skrev popeyeotaku ***@***.***>:

**Comment by jkotlinski** [comment]
I noticed now there was some relevant feedback for v5.0.0 in https://github.com/jkotlinski/durexforth/discussions/548 .
